be/src/olap/tablet_manager.cpp

```diff
@@ -1060,7 +1060,74 @@ void TabletManager::build_all_report_tablets_info(std::map<TTabletId, TTablet>*
     LOG(INFO) << "success to build all report tablets info. tablet_count=" << tablets_info->size();
 }
 
+Status TabletManager::try_move_to_trash(TTabletId tablet_id, TSchemaHash schema_hash,
+                                        const io::Path& tablet_path) {
+    TabletSharedPtr tablet = nullptr;
```

Comment:
use vector<tablet>  because the maybe multiple tablets with the same tablet_id、schema_hash、data_dir.